js/src/modal.js

```diff
@@ -418,21 +418,36 @@ const Modal = (($) => {
     }
 
     _setScrollbar() {
-      let bodyPadding = parseInt(
-        $(Selector.FIXED_CONTENT).css('padding-right') || 0,
-        10
-      )
-
-      this._originalBodyPadding = document.body.style.paddingRight || ''
-
       if (this._isBodyOverflowing) {
+        // Adjust Fixed Content Padding
+        $(Selector.FIXED_CONTENT).map((index, element) => {
+          let fixedPadding = $(element).css('padding-right')
+          this._originalPadding[index] = fixedPadding
```

Comment:
Store the original padding in a `data-*` attribute on each element instead of in `_originalPadding`